manifests/m/Microsoft/DSC/3.1.0-preview.1/Microsoft.DSC.locale.en-US.yaml

```
@@ -0,0 +1,41 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Microsoft.DSC

```

```suggestion
PackageIdentifier: Microsoft.DSC.Preview
```

This is a preview version that should be added under `Microsoft.DSC.Preview` Package ID